packages/block-library/src/navigation/view.js

```diff
@@ -1,35 +1,33 @@
 /**
  * WordPress dependencies
  */
-import { store } from '@wordpress/interactivity';
+import { store as wpStore } from '@wordpress/interactivity';
 
 const focusableSelectors = [
 	'a[href]',
-	'area[href]',
 	'input:not([disabled]):not([type="hidden"]):not([aria-hidden])',
 	'select:not([disabled]):not([aria-hidden])',
 	'textarea:not([disabled]):not([aria-hidden])',
 	'button:not([disabled]):not([aria-hidden])',
-	'iframe',
-	'object',
-	'embed',
```

Comment:
I removed the elements that are unnecessary for the menu. If in the future we need them, we can add them back.